index.js

```diff
@@ -650,3 +657,10 @@ function getClosingParenthensePosition (path, idx) {
 
   throw new TypeError('Invalid regexp expression in "' + path + '"')
 }
+
+function defaultBuildPrettyMeta (route) {
+  // buildPrettyMeta function must return an object, which will be parsed into key/value pairs for display
+  if (!route) return {}
+  if (!route.store) return {}
+  return Object.assign({}, route.store)
```

Comment:
Is this shallow copy needed?

Reply:
It seems like a good idea since, specifically in the `fastify` use case, we're exposing some properties that are private and could break things if they're mutated. A deep copy felt like overkill since the function can be overridden by the user.

Reply:
> Is not it a shortcut to skip the `buildPrettyMeta` function?

No, the route is forced to go through the `buildPrettyMeta` function before it's displayed. If a user needed to bypass the function, they could override it and just return the original `route` object with no processing.